src/fetchers.js

```diff
@@ -304,9 +304,10 @@ async function resolveRef(opts, log) {
         branch: ref,
       };
     }
-    log.error(`Unable to resolve branch name ${res.statusCode} ${res.body}`);
+    const level = (!res.statusCode || res.statusCode >= 500) ? 'error' : 'info';
```

Comment:
This looks like a hack that is a bit too clever – the most likely reason is that you want to keep the line coverage up by avoiding an if statement, so that you don't have to add a new test case.

Reply:
Actually, that wasn't the reason 😉 I just wanted to keep the footprint as small as possible. And the exact same "hack" already got approved by @tripodsan in https://github.com/adobe-private/helix-googledocs2md/pull/87.

But if you prefer, I can add an `if` and of course a test.

Reply:
I would prefer, but don't insist on, that.

Reply:
I'm ok with that....we could also mandate the 100% statement coverage, then the syntax doesn't matter.